Dockerfile.tools

```diff
@@ -15,7 +15,7 @@ RUN ["make", "service-build"]
 CMD ["make", "service-start"]
 
 # Production
-FROM mongo:4.2.14 AS production
+FROM mongo:6.0.5 AS production
```

Comment:
Latest mongodb 6.x uses mongosh 1.8, which the 5.0 repos we use on jump boxes are also using, I tested by raising the relevant QA jumpbox to 6.

If we want to test specifically from the tools pod, this is not currently deployed, but it can be.

Reply:
Updated wrapper scripts to handle dropped `mongo` shell for mongosh.

Reply:
Please deploy for testing